azurerm/internal/services/containers/resource_arm_kubernetes_cluster.go

```diff
@@ -1427,15 +1553,42 @@ func flattenKubernetesClusterNetworkProfile(profile *containerservice.NetworkPro
 		podCidr = *profile.PodCidr
 	}
 
+	lbProfiles := make([]interface{}, 0)
+	if profile.LoadBalancerProfile != nil {
```

Comment:
This would be a lot cleaner if we 
```suggestion
	if lbp := profile.LoadBalancerProfile; lbp != nil {
```